users/templates/users/profile.html

```diff
@@ -1,6 +1,23 @@
 {% extends "base.html" %}
 {% load crispy_forms_tags %}
 
+<style media="screen">
+  .table-responsive {
+    max-height:300px;
+}
+
+.wrapper {
+    position: relative;
+}
+
+.overlay th {
+    background-color: #212121;
+    color: white;
+    position: absolute;
+    top:0;
+}
+</style>
+
```

Comment:
To avoid repeating yourself, these should be put in a CSS file and imported into the html, although it doesn't really matter for now.
Could you accomplish the same thing with the [Bootstrap Overflow](https://getbootstrap.com/docs/4.3/utilities/overflow/) that have already imported?

Reply:
I am on it!